src/main/java/org/alfresco/repo/web/scripts/solr/TransactionIntervalGet.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * #%L
+ * Alfresco Repository
+ * %%
+ * Copyright (C) 2005 - 2019 Alfresco Software Limited
```

Comment:
2020

Reply:
Well, technically this was developed past year...
:)